tests/task_graph_test.py

```diff
@@ -31,7 +31,7 @@ def as_float(x: int) -> float:
 
 def make_task_graph() -> Graph:
     pl = sl.Pipeline([as_float], params={int: 1})
-    return pl.build(float, handler=sl.HandleAsBuildTimeException())
+    return pl.build((float,), handler=sl.HandleAsBuildTimeException())
```

Comment:
Is the interface of `build` and `get` different now? Why? This seems confusing.

Reply:
`build` is a lower-level method with a simpler interface. The complexity of distinguishing between single and multiple keys was moved in to `get`. `build` should (as before) not be used by users.